### PR TITLE
Add BigQuery schema conversion util

### PR DESCRIPTION
**Pull request by ANich**
Adds utils for converting from BigQuery schema to local `Schema` and vice-versa.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2757)

### CODE REVIEW

**Review by kwindau — APPROVED** [review]
lgtm

File: bigquery_etl/schema/__init__.py [hunk 1]
```diff
@@ -282,3 +283,14 @@ def to_json_file(self, json_path: Path):
     def to_json(self):
         """Return the schema data as JSON."""
         return json.dumps(self.schema)
+
+    def to_bigquery_schema(self) -> List[SchemaField]:
+        """Get the BigQuery representation of the schema."""
+        with NamedTemporaryFile(suffix=".json") as f:
+            self.to_json_file(Path(f.name))
+            return [SchemaField.from_api_repr(field) for field in json.load(f)]
```
Comment (sean-rose) [inline bigquery_etl/schema/__init__.py:291]: I see this is just moving/de-duping the code formerly in `query.py`, but it seems like the temp file isn't necessary.
```suggestion
        return [SchemaField.from_api_repr(field) for field in self.schema["fields"]]
```

**Review by sean-rose — APPROVED** [review]
r+wc

File: bigquery_etl/schema/__init__.py (hunk 1 above)
Comment (ANich) [reply bigquery_etl/schema/__init__.py:291]: This works yeah

**Pull request merged**